dist/manifest.json

```diff
@@ -33,7 +33,10 @@
       "https://superuser.com/*",
       "https://askubuntu.com/*",
       "http://stackoverflow.com/*",
-      "https://stackoverflow.com/*"
+      "https://stackoverflow.com/*",
+      "http://package.elm-lang.org/*",
```

Comment:
I tested on this site and for some reason it's not working. Maybe it's because they load the content with ajax which prevents the extension from working.

URL tested: http://package.elm-lang.org/packages/Bernardoow/elm-alert-timer-message/1.0.0/